RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -41,7 +37,7 @@
 #include "TrackingTools/PatternTools/interface/TrajectoryMeasurement.h"
 #include "TrackingTools/PatternTools/interface/TrajMeasLessEstim.h"
 
-class OutsideInMuonSeeder : public edm::stream::EDProducer<> {
+class OutsideInMuonSeeder final : public edm::stream::EDProducer<> {
```

Comment:
(For my own education on C++) if this class is declared final, what is the reason of having virtual methods in it?